doc/CloudIntegration.xml

```diff
@@ -0,0 +1,338 @@
+      </section>
+      <section xml:id="_Ref208379139">
+        <title>Security</title>
+        <section>
+          <title>Authentication</title>
+          <para>The services defined in this standard, whenever consumed, shall be protected using
+          digest authentication according to [RFC 2617] over HTTP and HTTPS or JWT client
+          authorization based on [RFC 6750], only over HTTPS. Additional information about HTTP
+          digest  authenticationcan can be found the ONVIF Core specification, while the JWT client
+          authorization can be found in the ONVIF Security Service.</para>
+          <para>MCS and OCS may support the SHA-256 hashing algorithm in HTTP and HTTPS. An ONVIF
+          compliant service should follow [RFC 7616].</para>
+        </section>
+        <section>
+          <title>User levels and access classes for the service</title>
+          <para>Since this service is only meant to enable sharing devices between different cloud
+          services, there is no need for defining different access classes to authorized access to
+          the functions defined in this specification.</para>
+        </section>
+      </section>
+    <section xml:id="_Ref208379149">
+      <title>Format of the share token</title>
+      <para>To start the device sharing operation, the OCS must provide a share token to the PCS, to
+        prove it has the rights to access the camera. They way the OCS retrieves the share token is
+        beyond the scope of this specification.</para>
+      <para>The share token must be a JWT and must include the <emphasis>xaddr</emphasis> claim that
+        will include the URI where the PCS will be accepting incoming requests from the OCS.</para>
+      </section>
+  </chapter>
+  <chapter>
+    <title>Device sharing</title>
+    <section>
+      <title>startDeviceSharing</title>
+      <para>This operation triggers sharing a device with a new Operational Cloud Service.</para>
+      <para>Once the information is sent to the camera, the camera will generate a public key
+        associated to the assigned ClientID. The generated public key will be passed to the PCS, so
+        that it will transfer it to the OCS with the deviceSharingCompleted function, so that it
+        will be able to authenticate the device and let it retrieve a JWT meant to be used to
+        authenticate the Uplink Service.</para>
+      <variablelist role="op">
+        <varlistentry>
+          <term>request</term>
+          <listitem>
+            <para role="text">startDeviceSharing [startDeviceSharing]</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>response</term>
+          <listitem>
+            <para role="param">This is an empty message</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>faults</term>
+          <listitem>
+            <para role="text">400 - Invalid Argument Value</para>
```

Comment:
Do we need to prepare anything related to device capabilities here? If the OCS tries to claim a device with settings that may not be supported by the device once it connects for example?
Here's we're thinking about the potentially moving target of the Security requirements, where an OCS could try to configure things that are not yet supported by the device, and would need to know about it somehow.

Reply:
Agreed @jmelancongen this was mentioned at the last F2F if I remember but not resolved.